test/util/forceDateToZero.js

```diff
@@ -1,6 +1,7 @@
 // need to standardize date to standardize log tests
 var OrigDate = Date;
 var testTime = 0;
+process.env.TZ = 'EST';
```

Comment:
Good, this was breaking the CI before. Will need to find a better solution, but this should be OK for now.